holoviews/plotting/bokeh/links.py

```diff
@@ -141,25 +141,30 @@ def __init__(self, root_model, link, source_plot, target_plot):
             if axis not in link.axes:
                 continue
 
-            axes[f'{axis}_range'] = target_plot.handles[f'{axis}_range']
+            range_name = f'{axis}_range'
+            if f'subcoordinate_{axis}_range' in target_plot.handles:
+                target_range_name = f'subcoordinate_{range_name}'
+            else:
+                target_range_name = range_name
+            axes[range_name] = target_plot.handles[target_range_name]
```

Comment:
```suggestion
            axes[range_name] = ax = target_plot.handles[target_range_name]
```

When we are already changing things, and then use `ax` for the rest of the code.